business_objects/embedding.py

```diff
@@ -327,6 +340,74 @@ def get_match_record_ids_to_qdrant_ids(
     return [r[0] for r in general.execute_all(query)]
 
 
+def get_match_record_ids_to_qdrant_ids_with_max_score(
+    project_id: str, embedding_id: str, qdrant_results: List[Any], limit: int
+) -> List[Any]:
```

Comment:
not sure if it makes sense but usually we have a default value for limit?

Reply:
not in this case as it's a required parameter calculated by the embedding service